src/lib/es6.d.ts

```diff
@@ -127,7 +127,7 @@ interface ObjectConstructor {
       * @param target The target object to copy to.
       * @param source The source object from which to copy properties.
       */
-    assign<T, U>(target: T, source: U): T & U;
+    assign<T, U>(target: T, source: U | null | undefined): T & U;

```

Comment:
Since this is brand new stuff, maybe @ahejlsberg should add a reduction rule from `T | null | undefined` to `T`... that may simplify a few other places where this might come up?